### PR TITLE
Add existence of hbase tables to the prereqs of opentsdb reader

### PR DESCRIPTION
**Pull request by youngklee**
Since writer/0 creates tables in hbase, if a reader starts before writer/0 does, it has to wait until writer/0 creates all tables in hbase.

### CODE REVIEW

**Comment by youngklee** [comment]
### Unit Test
```
(zenoss) [zenoss@84ab11eec6c9 migrate]$ runtests --type unit --name test_checkHBaseTablesExist Products.ZenModel.migrate
==============================

Packages to be tested:
        Products.ZenModel.migrate

==============================
Parsing /opt/zenoss/etc/zope.conf
Running tests at level 1
Running zope.testrunner.layer.UnitTests tests:
  Set up zope.testrunner.layer.UnitTests in 0.000 seconds.
  Running:
..
  Ran 2 tests with 0 failures and 0 errors in 1.335 seconds.
Tearing down left over layers:
  Tear down zope.testrunner.layer.UnitTests in 0.000 seconds.
(zenoss) [zenoss@84ab11eec6c9 migrate]$ 
```

### Run Migration
```
(zenoss) [zenoss@5ddeaf30eacb ~]$ zenmigrate --step CheckHBaseTablesExist 
No handlers could be found for logger "root"
2016-09-29 22:13:14 INFO zen.migrate: Will execute these steps: CheckHBaseTablesExist
2016-09-29 22:13:14 INFO zen.migrate: Installing CheckHBaseTablesExist (5.2.0)
2016-09-29 22:13:34 INFO zen.migrate: Committing changes
2016-09-29 22:13:34 INFO zen.migrate: Migration successful
(zenoss) [zenoss@5ddeaf30eacb ~]$
```

### Diff
```
ylee@zenoss-1310-d:~/src/metis/src/github.com/zenoss/zenoss-prodbin/Products/ZenModel/migrate/tests$ diff zenoss-resmgr-5.1.5.json zenoss-resmgr-5.1.5-checkHBaseTablesExist.json 
18155a18156,18159
>        },
>        {
>          "Name": "HBase tables exist",
>          "Script": "wget -q -O- http://localhost:61000 | [[ $(grep -c -E -o \"\\b${CONTROLPLANE_TENANT_ID}-tsdb(-|\\s|$)\") == 4 ]]"
ylee@zenoss-1310-d:~/src/metis/src/github.com/zenoss/zenoss-prodbin/Products/ZenModel/migrate/tests$  
```

File: Products/ZenModel/migrate/checkHBaseTablesExist.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+    def cutover(self, dmd):
+        try:
+            ctx = sm.ServiceContext()
+        except sm.ServiceMigrationError:
+            log.info("Couldn't generate service context, skipping.")
+            return
+
+        # Find the services to edit.
+        # For "lite" services, there is a single opentsdb service; edit that
```
Comment (iancmcc) [inline Products/ZenModel/migrate/checkHBaseTablesExist.py]: Definitely don't migrate lite services! They'll never start!

Comment (iancmcc) [inline Products/ZenModel/migrate/checkHBaseTablesExist.py]: Although it looks from your code like you aren't migrating them anyway, so just change the comment.

Comment (youngklee) [inline Products/ZenModel/migrate/checkHBaseTablesExist.py]: The comment has been updated.

**Pull request merged**